admin/data/parcelpro-shipping-settings-fields.php

```diff
@@ -125,6 +125,34 @@
         'description' => 'Customer note (billing address) invullen bij het veld "inhoud" in het verzendsysteen.',
         'desc_tip'    => true
     ),
+    'parcelpro_forecast_settings_postnl_show_delivery_date' => array(
+        'type'     => 'checkbox',
+        'title'    => 'Verwachte levertijd inschakelen (PostNl)',
+        'label'    => 'Verwachte levertijd inschakelen',
+        'desc_tip' => true,
+        'default'  => 'no',
+    ),
+    'parcelpro_forecast_settings_postnl_cutoff_time' => array(
+        'type'     => 'text',
+        'title'    => 'Cutoff tijd van PostNL voor verwachte levertijd berekening.',
+        'label'    => 'Cutoff tijd van PostNL voor verwachte levertijd berekening.',
+        'default'  => '17:00 (Europe/Amsterdam)',
```

Comment:
Ik zie in de Magento code dat ik daar "22:00 Europe/Amsterdam" (dus zonder de haakjes), en als ik lokaal test lijken de haakjes het ook te breken:

![image](https://github.com/parcel-pro-nl/woocommerce-plugin/assets/5592716/3d04cd30-bdd6-4eda-b67a-72db77d19c90)